semantic/src/site/modules/dropdown.variables

```diff
@@ -1,3 +1,5 @@
 /*******************************
     User Variable Overrides
 *******************************/
+
+@selectionBorder: 1px solid #777777;
```

Comment:
Yeah, I tried using `@headersInteractiveLow` but it seemed too dark to me in this context. Should we make a new custom colour for "outlines" (since I think you'll need the same one for the Segment borders in Review Home page)?

Reply:
Yep. That's a good idea.

Reply:
Cool, since you've also used this for the Segment borders in ReviewHome, I'll wait till you've merged that into here, then I'll replace them all with the variable value.

Reply:
Nice. Will do that now.